src/nested_pandas/series/accessor.py

```diff
@@ -540,3 +540,94 @@ def clear(self) -> None:
         The reason is that we cannot delete all nested fields from the nested series.
         """
         raise NotImplementedError("Cannot delete fields from nested series")
+
+    def to_flatten_inner(self, field: str) -> pd.Series:
+        """Explode the nested inner field and return as a pd.Series
+
+        Works for the case of multiple nesting only, the field must represent
+        a nested series.
+
+        Each row of this Series is changed in the following way:
+        1. Each nested item in the given field is converted to a "flat" frame.
+        2. All items of other fields are repeated as many times as that frame
```

Comment:
Is the expectation here that this will only work for doubly nested structures and not triply+ nested structures

Reply:
This works to flat the second level only, the third level will become the second level after that. Could you please suggest a docstring improvement?..